Yoast/javascript
# Refactor integration

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the necessary steps to integrate the new yoastseo structure with the plugin.


## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* See accompanying [plugin PR](https://github.com/Yoast/wordpress-seo/pull/16619) for test steps.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-652
